rosa_architecture/index.adoc

```diff
@@ -59,7 +59,7 @@ Start with xref:../architecture/architecture.adoc#architecture-overview-architec
 xref:../security/container_security/security-understanding.adoc#understanding-security[Security and compliance].
 ifdef::openshift-enterprise,openshift-webscale[]
 Next, view the
-xref:../release_notes/ocp-4-15-release-notes.adoc#ocp-4-15-release-notes[release notes].
+xref:../release_notes/ocp-release-notes.adoc#ocp-4-15-release-notes[release notes].
```

Comment:
It's a pity that an attribute does not exist at the global level for 4-15. Kind of like how {product-version}. Would you know if it's possible to create such as attribute?

Reply:
I've a PR in motion to add the 4.16 and 4.17 stubs to the `main` branch: https://github.com/openshift/openshift-docs/pull/81083 . I've updated RN checklist to reflect this is a required task in future releases.